source/algorithms.tex

```diff
@@ -846,11 +846,13 @@
 
   namespace ranges {
     template<forward_iterator I, sentinel_for<I> S, class Proj = identity,
-             indirect_relation<projected<I, Proj>> Pred = ranges::equal_to>
+             indirect_binary_predicate<projected<I, Proj>,
+                                       projected<I, Proj>> Pred = ranges::equal_to>
```

Comment:
> (There might be arcane cases where users might want to specify explicit template argument lists.)

[[algorithms.requirements]p15](http://eel.is/c++draft/algorithms.requirements#15):
> The number and order of deducible template parameters for algorithm declarations are unspecified, except where explicitly stated otherwise.
[ Note: Consequently, the algorithms may not be called with explicitly-specified template argument lists. — end note ]